invenio_edugain/templates/semantic-ui/invenio_edugain/login_discovery.html

```diff
@@ -0,0 +1,53 @@
+{# -*- coding: utf-8 -*-
+
+  Copyright (C) 2025 Graz University of Technology.
+
+  invenio-edugain is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% extends config.INVENIO_THEME_TUGRAZ_ACCOUNT_BASE %}
```

Comment:
the package has to be independent of *_TUGRAZ_*

Reply:
Replaced with the vanilla account base `config.ACCOUNTS_BASE_TEMPLATE`